src/app/layout.tsx

```diff
@@ -17,7 +25,9 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={`${dm_mono.className} bg-background text-foreground`}>
+      <body
+        className={`${dm_mono.className} ${syne.variable} bg-brand-midnight text-foreground`}
+      >
```

Comment:
Au lieu de faire des template literals pour les classes tailwind, tu peux utiliser `cn()`. 

```suggestion
        className={cn(dm_mono.className, syne.variable, "bg-brand-midnight text-foreground"}
      >
```